File: aula 2/main.js
class Usuario {
    constructor(name, email, idade) {
        this.nome = name
        this.email = email
        this.idade = idade
    }

    exibirPerfil = () => {
        console.log(`Usuário: ${this.nome} | Idade: ${this.idade} | Contato: ${this.email}`);
    }

    aumentaIdade = () => {
        ++this.idade
    }
}

const pessoa1 = new Usuario("Jhean Monteiro", "Jhean@email.com", 21)
const pessoa2 = new Usuario("Pedro Monteiro", "Pedro@email.com", 25)
pessoa2.aumentaIdade()

pessoa1.exibirPerfil()
pessoa2.exibirPerfil()

File: aula 2/main.py
class Usuario:
    def __init__(self, nome, email):
        self.nome = nome
        self.email = email

    def exibir_perfil(self):
        print(f"\nUsuario: {self.nome} | Contato: {self.email}\n")

pessoa1 = Usuario("Jhean", "jhean@example.com")
pessoa2 = Usuario("Pedro", "pedro@example.com")

pessoa1.exibir_perfil()
pessoa2.exibir_perfil()